src/components/Header/index.js

```diff
@@ -16,41 +16,51 @@ import List from 'material-ui-icons/List'
 import Search from 'material-ui-icons/Search'
 import Add from 'material-ui-icons/Add'
 import HelpOutline from 'material-ui-icons/HelpOutline'
+import Person from 'material-ui-icons/Person'
 
 import {Link} from 'react-router-dom'
 
-import cityOfHelsinkiLogo from 'src/assets/images/helsinki-coat-of-arms-white.png'
+import cityOfHelsinkiLogo from 'src/assets/images/helsinki-logo.1.svg'
 
 class HeaderBar extends React.Component {
 
     render() {
         let buttonStyle = {color: '#ffffff'}
+        let buttonStyle2 = {color: '#000000'}
+        let navstyle = {
+            width: '100%',
+            display: 'grid',
+            'grid-template-columns': '20% 60% 20%',
```

Comment:
have 20% and 60% in local file sass variable 
$headerSpan: 20%;
$headerLinkSpan: 60%;